packages/@aws-cdk/aws-iot-actions/test/sqs/sqs-queue-action.test.ts

```diff
@@ -0,0 +1,140 @@
+import { Template, Match } from '@aws-cdk/assertions';
+import * as iam from '@aws-cdk/aws-iam';
+import * as iot from '@aws-cdk/aws-iot';
+import * as sqs from '@aws-cdk/aws-sqs';
+import * as cdk from '@aws-cdk/core';
+import * as actions from '../../lib';
+
+test('Default sqs queue action', () => {
+  // GIVEN
+  const stack = new cdk.Stack();
+  const topicRule = new iot.TopicRule(stack, 'MyTopicRule', {
+    sql: iot.IotSql.fromStringAsVer20160323("SELECT topic(2) as device_id FROM 'device/+/data'"),
+  });
+  const queue = sqs.Queue.fromQueueArn(stack, 'MyQueue', 'arn:aws:s3::123456789012:test-queue');
+
+  // WHEN
+  topicRule.addAction(
+    new actions.SQSQueueAction(queue),
+  );
+
+  // THEN
+  Template.fromStack(stack).hasResourceProperties('AWS::IoT::TopicRule', {
+    TopicRulePayload: {
+      Actions: [
+        {
+          Sqs: {
+            QueueUrl: {
+              'Fn::Join': [
+                '',
+                [
+                  'https://sqs..',
+                  {
+                    Ref: 'AWS::URLSuffix',
+                  },
+                  '/123456789012/test-queue',
+                ],
+              ],
```

Comment:
Let's try to preserve vertical space here:

```suggestion
              'Fn::Join': ['', [
                'https://sqs..',
                { Ref: 'AWS::URLSuffix' },
                '/123456789012/test-queue',
              ]],
```